CONTRIBUTING.md

```diff
@@ -2,16 +2,29 @@
 We want to make contributing to this project as easy and transparent as
 possible.
 
-## Pull Requests
+
+
+## Notes on Pull Requests
 We actively welcome your pull requests.
 
-1. Fork the repo and create your branch from `master`.
+1. Fork the repo and create your branch from `main`.
 2. If you've added code that should be tested, add tests.
 3. If you've changed APIs, update the documentation.
 4. Ensure the test suite passes.
 5. Make sure your code lints.
 6. If you haven't already, complete the Contributor License Agreement ("CLA").
-7. We have adopted squash-and-merge as the policy for incorporating PRs into the master branch.  We encourage more smaller/focused PRs rather than big PRs with many independent changes.  This also enables faster development by merging PRs into master quickly and reducing the need to rebase due to changes on master.
+7. We have adopted squash-and-merge as the policy for incorporating PRs into the `main` branch.
+8. We would rather have several smaller pull requests that a single large Pull Request since smaller self-contained changes are easier to review.
+
+We accept two types of Pull Requests. Make sure your Pull Requests are not a mix of these two types.
+
+**Development Pull Requests** that add new features to the [habitat-lab](/habitat-lab) task and environment codebase. Development Pull Requests must be small, have unit testing, very extensive documentation and examples. These are typically new tasks, environments, sensors, etc... The review process for these Pull Request is longer because these changes will be maintained by our core team of developers, so make sure your changes are easy to understand!
+
+**Experiments Pull Requests** that add new features to the [habitat-baselines](/habitat-baselines/) training codebase. Experiments Pull Requests can be any size, must have smoke/integration tests and be isolated from the rest of the code. Your code additions should not rely on other habitat-baselines code. This is to avoid dependencies between different parts of habitat-baselines. You must also include a README that will document how to use your new feature or trainer. **You** will be the maintainer of this code, if the code becomes stale and is not supported often enough, we will eventually remove it.
```

Comment:
Are we then accepting any work using habitat, as long as they follow the guidelines? Or we go for works we are developing internally, we could use for our projects.

Reply:
I think (up for debate once again) that we should accept any trainer/feature that uses habitat-lab in habitat-baselines. And we should also develop our own internally, especially DDPPO which is our starting point in many projects.